presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -2869,6 +2869,22 @@ public void testCreateAndInsert()
         assertEqualsIgnoreOrder(actualAfterTransaction, expected);
     }
 
+    @Test
+    public void testRenameView()
+    {
+        assertUpdate("CREATE VIEW rename_view_original AS SELECT COUNT(*) as count FROM orders");
```

Comment:
use a smaller table (or `SELECT 1`) as view body

Reply:
I'd rather use a query that does something... this test only takes 0.5 seconds on my laptop.